scripts/build.ts

```diff
@@ -322,36 +365,42 @@ async function buildJS() {
 		input[`plugins/prism-${id}`] = path.join(SRC_DIR, `plugins/${id}/prism-${id}.ts`);
 	}
 
-	let bundle;
+	// Using multiple bundles here mostly for convenience.
+	let esmBundle;
+	let cjsBundle;
```

Comment:
Non-blocking: 
1. We should also have a global bundle (that works similarly to v1, minus any deliberate API changes). 
2. Nit: especially once we have 3 bundles, using a data structure instead of variables might end up keeping things a bit more tidy.

Reply:
> 1. We should also have a global bundle (that works similarly to v1, minus any deliberate API changes).

I played with it a bit. It involves some changes to the way the Autoloader plugin is used. I'd suggest leaving it for a separate PR. We have a few things to discuss there.